src/org/usfirst/frc4904/robot/Robot.java

```diff
@@ -72,6 +76,13 @@ public void autonomousExecute() {}
 	@Override
 	public void alwaysExecute() {
 		putSDSubsystemSummary();
+		try {
+			SmartDashboard.putNumber("Boiler Angle Error", RobotMap.Component.lidar.getLidarSensor2().readSensor()[0]);
+		}
+		catch (InvalidSensorException e) {
+			SmartDashboard.putNumber("Boiler Angle Error", 180.0);
+		}
+		SmartDashboard.putNumber("Gear Angle Error", RobotMap.Component.gearAlignCamera.getDegrees());
```

Comment:
Make sure to use `SmartDashboardKey` for all of these instead of a magic string.